permissions/plugin-aws-sqs.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```

Comment:
@nickgrealy We strongly prefer artifactIds that do not have the `-plugin` suffix. See https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Prerequisites